google-cloud-spanner/src/test/java/com/google/cloud/spanner/AbstractStructReaderTypesTest.java

```diff
@@ -301,6 +312,13 @@ public static Collection<Object[]> parameters() {
             "getDate",
             Collections.singletonList("getValue")
           },
+            {
+                Type.uuid(),
+                "getUuidInternal",
```

Comment:
not: formatting not consistent on the class

Reply:
Done.